src/endpoints/genomicFeatureSuggestions.js

```diff
@@ -1,18 +1,25 @@
 import EsInstance from '../ElasticSearchClientInstance';
-import { indexNameGenomicFeatureSuggestion, maxNOfGenomicFeatureSuggestions } from '../env';
+import {
+  indexNameGeneFeatureSuggestion,
+  indexNameVariantFeatureSuggestion,
+  maxNOfGenomicFeatureSuggestions,
+} from '../env';
 import { StatusCodes } from 'http-status-codes';
+import { suggestionType } from '../app';
 
-const NEEDED_SOURCE_FIELDS = ['type', 'suggestion_id', 'locus', 'symbol'];
-
-export default async (req, res) => {
+export default async (req, res, _suggestionType) => {
   const prefix = req.params.prefix;
 
   const client = await EsInstance.getInstance();
 
+  const _index =
+    _suggestionType === suggestionType.GENE
```

Comment:
why not naming the enum SUGGESTIONS_TYPES and do and _suggestionType to type (we know from the context that we are talking about suggestions)

suggestionType = SUGGESTIONS_TYPES[type]